source/jormungandr/tests/journey_common_tests.py

```diff
@@ -34,35 +34,683 @@
 from .check_utils import *
 
 
+def check_journeys(resp):
+    assert not resp.get('journeys') or sum([1 for j in resp['journeys'] if j['type'] == "best"]) == 1
```

Comment:
since you're now an expert of python, let's use generator all the time   :wink:
`sum((1 for j in resp['journeys'] if j['type'] == "best"))`